docs/.eslintrc.js

```diff
@@ -1,29 +1,104 @@
 module.exports = {
   root: true,
-  parser: 'vue-eslint-parser',
-  plugins: ['gridsome'],
-  env: {
-    node: true,
-  },
+  plugins: ['prettier', 'node', 'jest', 'jsdoc'],
   extends: [
-    'plugin:gridsome/recommended',
```

Comment:
same here, do you have any specific reason for removing gridsome specifics extends?

Reply:
Same above.